### PR TITLE
 (Fix): Copyright Aligment

### PR DESCRIPTION
**Pull request by dobri1408**
This feature is not working yet

### CODE REVIEW

**Comment by andreiggr** [comment]
https://user-images.githubusercontent.com/44702393/214830127-0380d1c3-4428-49b7-9f60-1448840d0c19.mp4




mobile big copyright

**Review by avoinea — CHANGES_REQUESTED** [review]
@andreiggr Doesn't work well with the latest changes in design-system / theme

**Comment by andreiggr** [comment]
image

Fixed copyright positioning when image is aligned full-width to align with content container

File: src/customizations/volto/components/manage/Blocks/Image/style.less [hunk 1]
```diff
@@ -1,12 +1,126 @@
...
+
+#copyright-hovering-text-inactive-right {
+  display: none;
+}
+
+#copyright-hovering-text-inactive-left {
+  display: none;
+}
+
+#copyright-hovering-text-active-right {
```
Comment (ichim-david) [inline src/customizations/volto/components/manage/Blocks/Image/style.less:98]: @andreiggr  don't use id's .. there will be more than one image
with copyright info thus you would get duplicate id's

File: src/customizations/volto/components/manage/Blocks/Image/style.less [hunk 2]
```diff
@@ -1,12 +1,126 @@
...
+  width: 10%;
+}
+
+.block.align.left .imageElement {
+  margin-right: 1em;
+  margin-left: 0;
+  float: left;
+}
+
+.block.align.left .large {
```
Comment (ichim-david) [inline src/customizations/volto/components/manage/Blocks/Image/style.less:56]: @andreiggr why do you need image css? shouldn't there be already code that sizes the images
when using the align classes?

File: src/customizations/volto/components/manage/Blocks/Image/components/Image.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  const classes = cx({
+    imageElement: true,
+    'full-width': align === 'full',
+    large: size === 'l',
+    medium: size === 'm',
+    small: size === 's',
+  });
+
+  return (
+    <div className={classes} style={styles}>
```
Comment (ichim-david) [inline src/customizations/volto/components/manage/Blocks/Image/components/Image.js:33]: @andreiggr don't use inline styles, our website theme will be used by thematic websites as well
and they might need to tweak some CSS styles

**Review by ichim-david — CHANGES_REQUESTED** [review]

File: src/customizations/volto/components/manage/Blocks/Image/components/Image.js (hunk 3 above)
Comment (andreiggr) [reply src/customizations/volto/components/manage/Blocks/Image/components/Image.js:44]: Hi @ichim-david 

Here inline styles are used to set the background image of that div. This way of setting the image is more ok for positioning the copyright properly.

File: src/customizations/volto/components/manage/Blocks/Image/style.less (hunk 2 above)
Comment (andreiggr) [reply src/customizations/volto/components/manage/Blocks/Image/style.less:56]: they're used for floating the div with the image. That code I think it targets img elements and that changed to enable the copyright correct positioning.

**Comment by avoinea** [comment]
Closing in favor of #92

**Pull request closed without merging**